inference-engine/tests/functional/plugin/gna/pass_tests/padded2valid_conv.cpp

```diff
@@ -0,0 +1,307 @@
+        auto bias_const = builder::makeConstant(ngPrc, bias_shape, bias_weights, true);
+        std::shared_ptr<Node> last_op = std::make_shared<Transpose>(conv, transpose_out_order);;
+
+        switch (model) {
+        case modelType::TranspConvBcastAddTransp:
+        {
+            auto bias = std::make_shared<Add>(conv, bias_const);
+            last_op = std::make_shared<Transpose>(bias, transpose_out_order);
+        }
+        break;
+
+        case modelType::TranspConvBcastAddMaxPoolTransp:
+        {
+            auto bcast_add = std::make_shared<Add>(conv, bias_const);
+            auto maxpool = std::make_shared<MaxPool>(bcast_add, Strides{ 1, 1 }, Shape{ 0, 0 }, Shape{ 0, 0 }, maxpool_shape);
+            last_op = std::make_shared<Transpose>(maxpool, transpose_out_order);
+        }
+        break;
+
+        case modelType::TranspConvBcastAddActTransp:
+        {
+            auto bcast_add = std::make_shared<Add>(conv, bias_const);
+            auto activation = std::make_shared<Relu>(bcast_add);
+            last_op = std::make_shared<Transpose>(activation, transpose_out_order);
+        }
+        break;
+
+        case modelType::TranspConvBcastAddMaxPoolActTransp:
+        {
+            auto bcast_add = std::make_shared<Add>(conv, bias_const);
+            auto max_pool = std::make_shared<MaxPool>(bcast_add, Strides{ 1, 1 }, Shape{ 0, 0 }, Shape{ 0, 0 }, maxpool_shape);
+            auto activation = std::make_shared<Relu>(max_pool);
+            last_op = std::make_shared<Transpose>(activation, transpose_out_order);
+        }
+        break;
+
+        case modelType::TranspConvTranspBcastAdd:
+        {
+            bias_const = std::make_shared<Constant>(ngPrc, transp_bias_shape);
+            last_op = std::make_shared<Add>(last_op, bias_const);
+        }
+        break;
+
+        case modelType::TranspConvTranspBcastAddAct:
+        {
+            bias_const = builder::makeConstant(ngPrc, transp_bias_shape, bias_weights, true);
+            auto bcast_add = std::make_shared<Add>(last_op, bias_const);
+            last_op = std::make_shared<Relu>(bcast_add);
+        }
+        break;
+
+        case modelType::TranspConvTransp:
+        default:
+            break;
+        }
+
+        function = std::make_shared<Function>(NodeVector{ last_op }, ParameterVector{ input });
+    }
+};
+
+class GnaPadded2ValidConvTest : public Padded2ValidConvTest, GnaLayerTestCheck {
+protected:
+    void Run() override {
+        GnaLayerTestCheck::SkipTestCheck();
+
+        if (!GnaLayerTestCheck::skipTest) {
+            Padded2ValidConvTest::Run();
+        }
+    }
+
+    void SetUp() override {
+        Padded2ValidConvTest::SetUp();
+    }
+};
+
+TEST_P(Padded2ValidConvTest, CompareWithRefs) {
+    Run();
+}
+
+TEST_P(GnaPadded2ValidConvTest, CompareWithRefs) {
+    Run();
+}
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32,
+    //InferenceEngine::Precision::FP16
```

Comment:
why is it commented?

Reply:
Some tests are failing here as well, I'll add a proper comment.